lib/kintsugi/apply_change_to_project.rb

```diff
@@ -1022,6 +1020,7 @@ def add_attributes_to_component(component, change, change_path, ignore_keys: [])
         next if (%w[isa displayName] + ignore_keys).include?(change_name)
 
         attribute_name = attribute_name_from_change_name(change_name)
+        child_path = join_path(change_path, attribute_name)
```

Comment:
I'm confused, you said you don't want to use attribute_name here, no?
Then again, if you don't use `attribute_name` here I don't understand what this PR fixes

Reply:
you're right, I changed it too many times and got confused 🤦

Reply:
This PR fixes the issue by not using the display name. It doesn't matter if you use the child name (camel case) or the attribute name (snake case), both are correct and converted later to snake case when the path is used.